front/lib/models/assistant/group_agent.ts

```diff
@@ -0,0 +1,91 @@
+import type {
+  BelongsToGetAssociationMixin,
+  CreationOptional,
+  ForeignKey,
+  InferAttributes,
+  InferCreationAttributes,
+} from "sequelize";
+import { DataTypes, Model } from "sequelize";
+
+import { AgentConfiguration } from "@app/lib/models/assistant/agent";
+import { frontSequelize } from "@app/lib/resources/storage";
+import { GroupModel } from "@app/lib/resources/storage/models/groups";
+
+export class GroupAgentModel extends Model<
```

Comment:
the modela must extend WorkspaceAwareModel and have the workspaceId (required for relocation - all models must have it)